apps/tup-cms/src/taccsite_cms/settings_custom.py

```diff
@@ -150,6 +150,35 @@
 
 TACC_SOCIAL_SHARE_PLATFORMS = ['linkedin', 'facebook', 'email']
 
+########################
+# TACC: EMAIL
+########################
+
+PORTAL_SHOULD_SEND_CONF_EMAIL = True
+TEXT_CONFIRMATION_EMAIL = """
```

Comment:
Renamed in https://github.com/TACC/Core-CMS/pull/829.

```suggestion
PORTAL_CONF_EMAIL_TEXT = """
```

Reply:
Resolved via 9c7007f.